clar2wasm/src/standard/standard.wat

```diff
@@ -1182,6 +1179,25 @@
         )
     )
 
+    ;;
+    ;; Logical Not implementation
+    ;;
+    (type $not_func (func (param i32) (result i32)))
+    (func $not (type $not_func)
+        (i32.eqz (local.get 0))
+    )
+    (export "not" (func $not))
+
+    ;;
+    ;; Xor implementation
+    ;;
+    (type $xor_func (func (param i64) (param i64) (param i64) (param i64) (result i64 i64)))
```

Comment:
You can reuse the existing `bit-xor` function instead of adding a new one.

Reply:
Done

Reply:
I still see `xor_func` in standard.wat.

Reply:
By the way, it's not documented anywhere, but the naming convention we've used for those functions is kebab-case, with the name matching the clarity expression, followed by the type, if needed, e.g. `add-uint`.

Reply:
This the "type" signature of the function... the function name is `xor`. Do you wanna to remove the function type?
And it is using kebab-case already... 🤔 Are we seeing the same thing? 😅

Reply:
Yeah, sorry, see my latest comments in the review I just submitted.